### PR TITLE
create tabs documentation

### PR DESCRIPTION
**Pull request by Mady-Camara**

### CODE REVIEW

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor [hunk 1]
```diff
@@ -1,6 +1,96 @@
...
+            Tabs are components that help you organize and navigate multiple documents in a single container. They can be used for switching between items in the container.
+        </p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            There are two pieces to a tabbed interface: the tabs themselves, and the content for each tab. The tabs are an component with the <Code Content="PFRhYkhlYWRlcj4="></Code>, and each item has the <Code Content="PFRhYkhlYWRlclRpdGxlPg=="></Code>. Each tab contains a link to a tab.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:16]: are an component ->  are a component
See [when to use a/an](https://www.grammar-monster.com/lessons/an_or_a.htm#:~:text=Using%20%22An%22%20and%20%22A%22,-There%20is%20sometimes&text=(The%20words%20%22an%22%20and,a%20house%20in%20an%20hour.)

and each item has the ??? the what the component?
Each tab contains a link to a tab. (Remove)

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor [hunk 2]
```diff
@@ -1,6 +1,96 @@
...
+        </p>
+        <p>
+            The tab content container has the <Code Content="PFRhYkNvbnRlbnRXcmFwcGVyPg=="></Code>, while each section has the <Code Content="PFRhYkNvbnRlbnQ+"></Code>. Each content pane also has a unique ID, which is targeted by a link in the tabstrip.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:19]: The tab content container has the ????? the what????
while each section has the ??????? same c'mon man!

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor [hunk 3]
```diff
@@ -1,6 +1,96 @@
...
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.TabContent>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.TabContentWrapper>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Vertical Tabs</h2>
+        <p>
+            Add the attribute <Code>Vertical=true</Code> to the <Code Content="PFRhYkhlYWRlcj4="></Code> and <Code>Class="vertical"</Code> to the <Code Content="PFRhYkNvbnRlbnRXcmFwcGVyPg=="></Code> to stack tabs vertically.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:46]: Add the attribute Vertical=true to the.... same thing
and Class="vertical" to the....

**Review by maxgsamson — CHANGES_REQUESTED** [review]
Well its good except some english stuff to change. Good job code wise but make sure to be toughoul

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor (hunk 1 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:16]: Yes 
```suggestion
            There are two pieces to a tabbed interface: the tabs themselves, and the content for each tab. The tabs are a component with the <Code Content="PFRhYkhlYWRlcj4="></Code>, and each item has the <Code Content="PFRhYkhlYWRlclRpdGxlPg=="></Code>. 
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor (hunk 2 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:19]: They’re in the Code Content

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor (hunk 3 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor:46]: They’re in the Code Content

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor (hunk 2 above)
Comment (maxgsamson) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor]: The English did not make sense

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor (hunk 3 above)
Comment (maxgsamson) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Tabs.razor]: The English did not make sense

**Pull request merged**